health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MobileChecklist.js

```
@@ -12,23 +12,43 @@ const MobileChecklist = ({ questions, checklistRole , typeOfChecklist}) => {
+                      {question.type.code === "SingleValueList" ? (
+                        <input
+                          type="radio"
+                          name={`question-${question.id}`}
+                          onClick={(e) => e.preventDefault()} // Prevents interaction
+                        />
+                      ) : (
+                        <input type="checkbox"
+                          onClick={(e) => e.preventDefault()} // Prevents interaction
+                        />
+                      )}
+                      <label className="mobile-answer">{option.label}</label>
```

Biome

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)